backend/dataall/modules/dataset_sharing/services/share_managers/s3_access_point_share_manager.py

```diff
@@ -26,10 +35,6 @@
 logger = logging.getLogger(__name__)
 ACCESS_POINT_CREATION_TIME = 30
 ACCESS_POINT_CREATION_RETRIES = 5
-DATAALL_ALLOW_OWNER_SID = 'AllowAllToAdmin'
-DATAALL_ACCESS_POINT_KMS_DECRYPT_SID = 'DataAll-Access-Point-KMS-Decrypt'
-DATAALL_KMS_PIVOT_ROLE_PERMISSIONS_SID = 'KMSPivotRolePermissions'
-DATAALL_DELEGATE_TO_ACCESS_POINT = 'DelegateAccessToAccessPoint'
 
```

Comment:
I think there are a couple of places where a share revoke could fail here. For example in `delete_dataset_bucket_policy` we call:
```
target_requester_arn = IAM.get_role_arn_by_name(
            self.target_account_id, self.target_environment.region, self.target_requester_IAMRoleName
        )
```
Which won't return anything, I would verify all calls to ` IAM.get_role_arn_by_name` in the data sharing functions

Reply:
Here it doesn't matter if it is None or not. The further actions are the same

Reply:
Issue adressed in [30b9d65](https://github.com/data-dot-all/dataall/pull/1161/commits/30b9d65b21969c27d2cea993bfd0bf2a86fa360d)